solidjs-tailwind/src/pages/Signin.jsx

```diff
@@ -0,0 +1,40 @@
+import { createEffect, createResource } from 'solid-js';
+import { useAuth } from '../auth';
+import { useNavigate } from '@solidjs/router';
+
+const fetchToken = () =>
+  fetch('http://localhost:4000/api/auth/token', {
```

Comment:
Can you make this an ebvironment variable, other showcases do it that way `http://api.starter.dev/api/`